akka-http-core/src/test/scala/akka/http/impl/engine/client/HighLevelOutgoingConnectionSpec.scala

```diff
@@ -91,5 +91,13 @@ class HighLevelOutgoingConnectionSpec extends AkkaSpec with ScalaFutures {
       a[One2OneBidiFlow.OutputTruncationException.type] should be thrownBy Await.result(x, 3.second)
       binding.futureValue.unbind()
     }
+
+    "the ticket 19287" in Utils.assertAllStagesStopped {
+      val r = Await.result(Http().singleRequest(HttpRequest(uri = "https://denner-api.detailnet.ch/screen/screen-publications")), 3.seconds)
```

Comment:
will need to remove that test though - no idea yet how to make one which we can simulate locally hm hm